### ISSUE TITLE
URL is not defined

### ISSUE THREAD

**Issue by tomitrescak** [thereactivestack/meteor-webpack#35]
Hi, when I updated to 0.4.0 am getting following error:

```
W20151215-13:18:25.070(1)? (STDERR) ReferenceError: URL is not defined
W20151215-13:18:25.070(1)? (STDERR)     at Object.eval (eval at <anonymous> (server.js:246:1), <anonymous>:1:18)
W20151215-13:18:25.070(1)? (STDERR)     at Object.<anonymous> (server.js:246:1)
W20151215-13:18:25.070(1)? (STDERR)     at __webpack_require__ (server.js:21:1)
W20151215-13:18:25.070(1)? (STDERR)     at Object.eval (eval at <anonymous> (server.js:240:1), <anonymous>:11:12)
W20151215-13:18:25.070(1)? (STDERR)     at Object.<anonymous> (server.js:240:1)
W20151215-13:18:25.070(1)? (STDERR)     at __webpack_require__ (server.js:21:1)
W20151215-13:18:25.070(1)? (STDERR)     at Object.eval (eval at <anonymous> (server.js:54:1), <anonymous>:44:1)
W20151215-13:18:25.070(1)? (STDERR)     at Object.<anonymous> (server.js:54:1)
W20151215-13:18:25.070(1)? (STDERR)     at __webpack_require__ (server.js:21:1)
W20151215-13:18:25.071(1)? (STDERR)     at Object.eval (eval at <anonymous> (server.js:48:1), <anonymous>:9:1)
```

**Comment by eXon** [comment]
Have you tried flushing your `.meetor/local/webpack-npm` folder?

**Comment by tomitrescak** [comment]
Yes, I reset the whole folder, deleted everything and let the NPM reinstall and rebuild everything ... ;( I am trying to localise what is the culprit ... but no luck. I reset the project back to previous commit all worked well, doing `meteor update webpack:webpack` led to the same error.

**Comment by tomitrescak** [comment]
@eXon I've found a culprit. Took some time. The problematic line in my code is `import * as url from "url";`

Since 0.4.0 I cannot use standard NPM packages. I tried even adding it to the webpack.packages.json but no luck. The only way to get rid of this error is to completely remove this line.

in my webpack.packages.json I have

```
  "url": "^0.11.0"
```

I even tried to flush the cache.

**Comment by eXon** [comment]
OK I'm pretty sure this is because Meteor has a URL export.

I'll fix Webpack so that import are prioritizing NPM modules before Meteor package exports.

**Comment by eXon** [comment]
Can you tell me if it is fixed in 0.4.1?

**Comment by tomitrescak** [comment]
Loks like it is fixed. Thanks!
